cluster/gce/delete-stranded-load-balancers.sh

```diff
@@ -14,22 +14,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# A utility for deleting target pools and forwarding rules that are unattached to VMs
+# A utility for deleting stranded load balancer resources in GCE
 PROJECT=${PROJECT:-kubernetes-jenkins}
 REGION=${REGION:-us-central1}
 
+# Deleting external load balancer resources
```

Comment:
This is external L4 right?

Reply:
yes. basically it's the gce resources created when user creates an external k8s LB.